tasks/utils/artifact.js

```diff
@@ -4,51 +4,54 @@ const fs = require("node:fs/promises")
 const currentVersion = "0"
 
 // export interface RequestData {
```

Comment:
Could also hold billing data - but keeping it leaner for now. If we want it later we can add to this structure and use the migration functions.

Reply:
@KuphJr Anything else we immediately want in here? If so, why?

Reply:
No, I think this is sufficient for now.